snapshot/doc.go

```diff
@@ -2,13 +2,12 @@
 package snapshot offers access to Scoot data.
 Scoot data is largely snapshots (hence the package name).
```

Comment:
Can a snapshot be anything other than a set of files?  (and I think of 'Scoot data' as a lot more than a snapshot - the term Scoot data in my mind includes the job definitions, saga data, snapshots, etc.)
Alternate definition:
'A Scoot snapshot is a set of files that will be made local to the environment in which a scoot command is run and/or it is the set of files that are preserved after a Scoot command is finished.'